hicommonwealth/commonwealth
# update community card with join and buy stake buttons and stake info

This PR updates the designs of `CWRelatedCommunityCard` to allow the user to join a community, buy stake from said community, and see the current price (and changes in the price) of stake.

Note that the conditional display of some parts of `CWRelatedCommunityCard` are to handle issues on the existing directory page. For instance, the display of stake has been made dependent on whether a stake value is present. That will be made dependent on whether stake is enabled as part of ticket #6869. In addition, `connected` will be a required prop. 

## Link to Issue
Closes: #6870

## Description of Changes
- adds a button to join the community
- adds a button to purchase stake
- displays information about stake

## Test Plan
- visit the old components page (`/components-old` path)
- confirm that the community card (toward the bottom of the page) matches those of the designs [here](https://www.figma.com/file/omZZrndm6Phbk1rOU4IKhS/Stake-V1-Fast-Follows?type=design&node-id=427-3588&mode=design&t=c3KJZHnOze0rGwqd-0)

## Other Considerations

- This card will be followed up by #6869, which places the cards on the explore communities page
